project3/index.html

```diff
@@ -0,0 +1,410 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Teko:300,400,500,600,700&display=swap" rel="stylesheet">
```

Comment:
1. These `link` references need to be inside the `head` below. Only `head` and `body` elements can be the direct children of the `html` element.
1. The font stylesheets can be combined as they are in [my example](https://github.com/mica-web/gd2/blob/1fa66b0aaa4ab516217ec9702adf905c198c187f/project3/index.html#L9).